project/overview.md

```diff
@@ -0,0 +1,37 @@
+<!-- vim: ft=markdown colorcolumn=72
+-->
+# Project Overview
+
+The spire/helm-chart effort intends to deliver a set of helm charts to
+support SPIRE deployments in Kubernetes to widest possible audience of
+users.
+
+## Mission and Scope
+This mission of this effort is to provide a stable upstream repository
```

Comment:
```suggestion
The mission of this effort is to provide a stable upstream repository
```

Reply:
Not a fan of referring to "this" before actually stating what this is in a concrete way.